tests/_schema/test_get_schema_definition.py

```diff
@@ -1,8 +1,19 @@
-from typedspark._core.datatypes import DayTimeIntervalType
-from typedspark._core.literaltype import IntervalType
+from typing import Annotated
```

Comment:
Maybe nice to also add some docs on this!

Reply:
I don't think it's necessary to have docs for this, it's a test that compares if two strings are the same